test/fixtures.yml

```diff
@@ -1291,6 +1291,9 @@ shift4:
   client_guid: YOUR_CLIENT_ID
   auth_token: YOUR_AUTH_TOKEN
 
+shift4_v2:
+  secret_key: pr_test_tXHm9qV9qV9bjIRHcQr9PLPa
```

Comment:
Can this environment variable be visible on the web?

Reply:
Yes is public info